Sources/ProjectDescription/BuildAction.swift

```diff
@@ -8,6 +8,8 @@ public struct BuildAction: Equatable, Codable, Sendable {
     public var preActions: [ExecutionAction]
     /// A list of actions that are executed after the build process.
     public var postActions: [ExecutionAction]
+    /// Whether Xcode should build targets in parallel based on dependency order
+    public var parallelizeBuild: Bool
```

Comment:
should we consider to name this property closer to what's in Xcode vs to how this is called under the hood? Additionally, should this be an enum instead of a bool to again better align with this property in Xcode?

I'm thinking of:
```swift
public var buildOrder: BuildOrder

public enum BuildOrder {
  case dependencyOrder
  case manualOrder
}
```